tribe-ext-community-currency-symbol/index.php

```diff
@@ -0,0 +1,402 @@
+<?php
+/**
+ * Plugin Name:     The Events Calendar Extension: Community Events Cost Currency Symbol
+ * Description:     Adds a currency symbol selector option to the Community Events form (must first be setup at wp-admin > Events > Settings > Community tab > Form Defaults section). Additionally, the event creator can override the currency symbol's position instead of using your site's default from wp-admin > Events > Settings > General tab.
+ * Version:         1.0.0
+ * Extension Class: Tribe__Extension__CE_Event_Cost_Currency_Symbol
+ * Author:          Modern Tribe, Inc.
+ * Author URI:      http://m.tri.be/1971
+ * License:         GPLv2 or later
+ * License URI:     https://www.gnu.org/licenses/gpl-2.0.html
+ */
+
+// Do not load unless Tribe Common is fully loaded.
+if ( ! class_exists( 'Tribe__Extension' ) ) {
+	return;
+}
+
+/**
+ * Extension main class, class begins loading on init() function.
+ */
+class Tribe__Extension__CE_Event_Cost_Currency_Symbol extends Tribe__Extension {
+
+	/**
+	 * Setup the Extension's properties.
+	 *
+	 * This always executes even if the required plugins are not present.
+	 */
+	public function construct() {
+		// Only works with/after version 4.5 as previous versions included the field
+		$this->add_required_plugin( 'Tribe__Events__Community__Main', '4.5' );
+
+		$this->set_url( 'https://theeventscalendar.com/extensions/community-events-cost-currency-symbol/' );
+	}
+
+	/**
+	 * Extension initialization and hooks.
+	 */
+	public function init() {
+		add_action( 'admin_init', array( $this, 'add_settings' ) );
+		add_action( 'tribe_events_community_section_before_cost', array( $this, 'display' ) );
+	}
+
+	/**
+	 * Add options to Tribe settings page.
+	 *
+	 * @see Tribe__Extension__Settings_Helper
+	 */
+	public function add_settings() {
+		require_once dirname( __FILE__ ) . '/src/Tribe/Settings_Helper.php';
+
+		// Setup fields on the settings page
+		$setting_helper = new Tribe__Extension__Settings_Helper();
+
+		// List of allowed symbols
+		$fields['tribe_ext_ce_cost_currency_symbols'] = array(
```

Comment:
You are implicitly declaring the `$fields` as an array here.  Normally it would explicitly declared first: `$fields = array()`. I'm not sure if this is necessary for backwards compatibility, or just how it's usually done. If you have tested this on PHP 5.2 then I'm sure implicit is fine. But if not, perhaps we should go the explicit route.